recipes/glfw/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanException
+
+
+class GlfwConan(ConanFile):
+    name = "glfw"
+    description = "GLFW is a free, Open Source, multi-platform library for OpenGL, OpenGL ES and Vulkan" \
+                  "application development. It provides a simple, platform-independent API for creating" \
+                  "windows, contexts and surfaces, reading input, handling events, etc."
+    settings = "os", "arch", "build_type", "compiler"
+    license = "Zlib"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/glfw/glfw"
+    topics = ("conan", "gflw", "opengl", "vulkan", "opengl-es")
+    generators = "cmake"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    _cmake = None
+
+    def _configure_cmake(self):
+        if not self._cmake:
+            self._cmake = CMake(self)
+            self._cmake.definitions["GLFW_BUILD_EXAMPLES"] = False
+            self._cmake.definitions["GLFW_BUILD_TESTS"] = False
+            self._cmake.definitions["GLFW_BUILD_DOCS"] = False
+            if self.settings.compiler == "Visual Studio":
+                self._cmake.definitions["USE_MSVC_RUNTIME_LIBRARY_DLL"] = "MD" in self.settings.compiler.runtime
+            self._cmake.configure(source_folder=self._source_subfolder)
+        return self._cmake
+
+    def system_requirements(self):
+        if tools.os_info.is_linux:
+            package_tool = tools.SystemPackageTool(conanfile=self, default_mode="verify")
```

Comment:
I think it would be better and easier to maintain if pkg-config is used instead of distro-specific package lists

Reply:
I agree

Reply:
+1